components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -384,7 +419,7 @@ def search_and_schedule_new_tasks(
         scheduled_jobs[job_id] = job
 
 
-def poll_running_jobs(db_conn, db_cursor):
+def poll_running_jobs(clp_config: CLPConfig, db_conn, db_cursor):
```

Comment:
Since we're only using `clp_config.logs_directory`, maybe we could just pass in `logs_directory`?